rllib/core/rl_trainer/tf/tf_rl_trainer.py

```diff
@@ -71,9 +70,13 @@ def apply_gradients(self, gradients: Mapping[str, Any]) -> None:
             optim.apply_gradients(zip(grad_groups, param_groups))
 
     @override(RLTrainer)
-    def _make_distributed(self) -> Tuple[RLModule, RLOptimizer]:
+    def _make_distributed(self) -> RLModule:
         # TODO: Does strategy has to be an attribute here? if so it's very hidden to
         # the user of this class that there is such an attribute.
+
+        # TODO (Kourosh, Avnish): The optimizers still need to be created within
```

Comment:
We need to unittest this then, since it was not properly caught in the previous PR.

Reply:
Not in this PR tho.